packages/fxa-content-server/server/lib/routes/post-metrics.js

```diff
@@ -152,6 +152,10 @@ const BODY_SCHEMA = {
   uniqueUserId: STRING_TYPE.regex(UNIQUE_USER_ID_PATTERN)
     .allow('none')
     .required(),
+  userPreferences: joi
```

Comment:
Can you do that now? We keep finding people throwing arbitrary garbage into the payloads to see what they can get away with.

Reply:
Whoops, meant to do that in this PR, missed my own TODO 🙇🏽‍♂️